build/check-translations.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs')
 const chalk = require('chalk')
 
 const defaultLangCode = 'en'
-const locations = ['modules/*/src/translations/*.json', 'src/bp/ui-*/src/**/translations/*.json']
+const locations = ['modules/**/translations/*.json', 'packages/bp/src/**/translations/*.json']
```

Comment:
Not sure if it matters here, but the packages `ui-admin` and `ui-shared` also have translations

Reply:
totally right. I'm on it